components/automate-ui/src/app/modules/user/user-management/user-management.component.ts

```diff
@@ -1,46 +1,28 @@
 import { Component, EventEmitter, OnInit, OnDestroy } from '@angular/core';
-import { FormBuilder, FormGroup, Validators } from '@angular/forms';
-import { Store, select } from '@ngrx/store';
+import { Store } from '@ngrx/store';
 import { Subject, combineLatest } from 'rxjs';
 import { takeUntil, filter } from 'rxjs/operators';
-import { isNil } from 'lodash/fp';
 
-import { LayoutFacadeService } from 'app/entities/layout/layout.facade';
 import { ChefSorters } from 'app/helpers/auth/sorter';
 import { NgrxStateAtom } from 'app/ngrx.reducers';
-import { Regex } from 'app/helpers/auth/regex';
-import { ChefValidators } from 'app/helpers/auth/validator';
-import { EntityStatus, pending } from 'app/entities/entities';
-import { allUsers, getStatus, createStatus, createError } from 'app/entities/users/user.selectors';
-import {
-  CreateUser, DeleteUser, GetUsers, CreateUserPayload
-} from 'app/entities/users/user.actions';
+import { EntityStatus } from 'app/entities/entities';
+import { LayoutFacadeService } from 'app/entities/layout/layout.facade';
+import { allUsers, getStatus } from 'app/entities/users/user.selectors';
+import { DeleteUser, GetUsers } from 'app/entities/users/user.actions';
 import { User } from 'app/entities/users/user.model';
-import { HttpStatus } from 'app/types/types';
-
-// pattern for valid usernames
-const USERNAME_PATTERN = '[0-9A-Za-z_@.+-]+';
 
 @Component({
   selector: 'app-user-management',
   templateUrl: './user-management.component.html',
   styleUrls: ['./user-management.component.scss']
 })
 export class UserManagementComponent implements OnInit, OnDestroy {
```

Comment:
Stripping out all the modal management from the users' list. Most of this moved into CreateUserModalComponent.